pennylane/qchem/molecule.py

```diff
@@ -55,6 +55,7 @@ class Molecule:
         r (array[float]): positions of the Gaussian functions
         normalize (bool): if True, the basis functions get normalized
         unit (str): unit of atomic coordinates. Available options are ``unit="bohr"`` and ``unit="angstrom"``.
+        argnum (int, list(int), None): differentiability of coords, coeffs, and alpha (in that order)
```

Comment:
```suggestion
        argnum (int | list(int) | None): index (indices) of the positional argument(s) - [``coordinates``, ``alpha``, ``coeff``] that should support differentiation. For example, ``argnums=[0, 2]`` would mean derivatives can be computed with respect to both ``coordinates`` and ``coeff``.
```
Also, please cross-check the order with the positional arguments.